server/src/lib/migration/migrations/v3-props.ts

```diff
@@ -43,6 +43,7 @@ function perToScore(goal: GoalDocument) {
 		case "bms":
 		case "gitadora":
 		case "jubeat":
+		case "maimai":
```

Comment:
This migration script is for moving existing scores (those stored on tachi 2.3) to 2.4. This will never be ran again, so it needs no update.

Reply:
the type checker isn't happy when i remove it

![image](https://user-images.githubusercontent.com/92439990/233434761-1448e9e5-d163-4221-9c38-3cfe2fee500c.png)

Reply:
ah, makes sense, even if it's nonsense and this migration has already ran.